src/pages/Photo/Photo.js

```diff
@@ -50,25 +51,23 @@ class Photo extends React.Component {
         return <span>There are no tags for this photo.</span>;
       if (data && data.tags[2] !== undefined) {
         const arr = Object.values(data.tags).map((tag) => {
-          return tag.title;
+          return <TagLink to={`/search/${tag.title}`}>{tag.title}</TagLink>;
```

Comment:
This whole function seems like it can be done better... I would take some time to rethink it

Reply:
how would you do it?

Reply:
Well first i would check if i have data when rendering the function, not here in every line.
Second i dont understand this in line 49: data.tags === 1, i believe data.tags is an array so what is she checking there? its length?

Reply:
Hmm...I will have to think about this one. I think I was supposed to be checking for length and just forgot to put it in. Good catch. Now I will just need to think about it's setup for a bit.

Reply:
What is the purpose of the function?

Reply:
To check that there are actually tags for the photo. I changed it to an if/else statement and it reads a bit better now. Still not sure if I'm completely satisfied, but also not sure how to make it better. 🤔